doc/version_history.rst

```diff
@@ -6,6 +6,12 @@
 Version History
 ###############
 
+v1.1.1
```

Comment:
I think these changes are too big for a patch release, consider making this a minor release, e.g. `v1.2.0`.